### PR TITLE
docs: update 4.3.1 changelog

### PR DESCRIPTION
**Pull request by LuckyPickleZZ**
#### What type of PR is this?
type-docs

#### What this PR does / why we need it:

update 4.3.1 changelog

#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: CHANGELOG-zh-CN.md [hunk 1]
```diff
@@ -1,5 +1,65 @@
...
+
+### 功能变化
+
+变更风险管控
+
+- 新增表级别权限管控，允许项目成员对不同表拥有不同操作权限，包括对表的查询、变更和导出操作
+
+会话管理
+
+- 当连接为 OceanBase 4.2.3 和 obproxy 4.2.5 或更高的版本时，使用 Client session 能力进行会话关闭
```
Comment (yizhouxw) [inline CHANGELOG-zh-CN.md:13]: feature side description: kill quey/sessson support more scenarios

File: CHANGELOG-zh-CN.md [hunk 2]
```diff
@@ -1,5 +1,65 @@
...
+- 在 OceanBase 4.2.1 oracle 模式下，使用匿名块进行会话关闭
+
+SQL 诊断
+
+- 支持以图格式查看 OceanBase 的逻辑 SQL 执行计划
```
Comment (yizhouxw) [inline CHANGELOG-zh-CN.md:18]: too simple, we do a lot of great job, it is desired to annount more details

File: CHANGELOG-zh-CN.md [hunk 3]
```diff
@@ -1,5 +1,65 @@
...
+- 诊断模块新增执行剖析功能，支持查看图形格式的实际 SQL 执行计划及实时执行数据
+
+### 易用性改进
+
+- SQL 执行时支持查看执行进度，并支持实时查看已完成的执行结果
```
Comment (yizhouxw) [inline CHANGELOG-zh-CN.md:23]: it is also a big feature for SQL Execution module, not an improvement

File: CHANGELOG-zh-CN.md [hunk 4]
```diff
@@ -1,5 +1,65 @@
...
+- 数据清理任务支持编辑任务配置
+- 数据源模块，支持批量导入 mysql、oracle 和 doris 数据源
```
Comment (yizhouxw) [inline CHANGELOG-zh-CN.md:25]: 支持批量导入 --> 批量导入支持

File: CHANGELOG-zh-CN.md (hunk 1 above)
Comment (LuckyPickleZZ) [reply CHANGELOG-zh-CN.md]: done

File: CHANGELOG-zh-CN.md (hunk 2 above)
Comment (LuckyPickleZZ) [reply CHANGELOG-zh-CN.md]: sure, I have expanded some of the content, please review if it is sufficient

File: CHANGELOG-zh-CN.md (hunk 3 above)
Comment (LuckyPickleZZ) [reply CHANGELOG-zh-CN.md]: done , moved it to `SQL Development` part

File: CHANGELOG-zh-CN.md (hunk 4 above)
Comment (LuckyPickleZZ) [reply CHANGELOG-zh-CN.md]: done

File: CHANGELOG-zh-CN.md [hunk 5]
```diff
@@ -1,5 +1,72 @@
...
+
+变更风险管控
+- 自动授权规则对 LoginSuccess 事件不生效 [#3003](https://github.com/oceanbase/odc/pull/3003)
+
+导入导出
+- 桌面版模式下，如果清理了元数据库但未清理工作空间，则再次发起导入任务时可能会存在脏数据文件 [#3006](https://github.com/oceanbase/odc/pull/3006)
+
+SQL 检查
+
+- 无法识别 `commit` 和 `rollback` 语句 [#2985](https://github.com/oceanbase/odc/pull/2985)
```
Comment (yizhouxw) [inline CHANGELOG-zh-CN.md:47]: 启用 SQL 窗口规范时，SQL 窗口的提交、回滚按钮可能会失效

File: CHANGELOG-zh-CN.md [hunk 6]
```diff
@@ -1,5 +1,72 @@
...
+
+模拟数据
+- 模拟数据任务无法终止 [#2850](https://github.com/oceanbase/odc/pull/2850)
+
+全局对象检索
+- 对象同步无法停止 [#2928](https://github.com/obase/odc/pull/2928)
+
+工单
+- 检查结果文件不存在本机时，无法获取 SQL 检查的结果 [#2943](https://github.com/oceanbase/odc/pull/2943)
+- 集群模型中的守护进程作业将同时被触发 [#2408](https://github.com/oceanbase/odc/pull/2408)
```
Comment (yizhouxw) [inline CHANGELOG-zh-CN.md:61]: 集群模型中的守护进程作业将同时被触发
>  it is hard for user to understand.

may: 多节点部署使用进程模式调度任务时，所有任务会被调度到同一个节点

and, it is not ticket module, may move to 其他

File: CHANGELOG-zh-CN.md [hunk 7]
```diff
@@ -1,5 +1,72 @@
...
+
+审计
+- `content_ip_address`列值的实际长度超过了该列的长度限制 [#2863](https://github.com/oceanbase/odc/pull/2863)
+
+其他
+- 反序列化分页对象失败 [#2434](https://github.com/oceanbase/odc/pull/2434)
```
Comment (yizhouxw) [inline CHANGELOG-zh-CN.md:67]: may remove this item, no specific bug behaivior here

File: CHANGELOG-zh-CN.md [hunk 8]
```diff
@@ -1,5 +1,72 @@
...
+
+会话管理
+
+- 关闭会话/查询支持更多场景
+  - 当连接为 OceanBase 4.2.3 和 obproxy 4.2.5 或更高的版本时，使用 Client session 能力进行会话管理
+  - 在 OceanBase 4.2.1 oracle 模式下，使用匿名块进行会话管理
+
+AP 开发
+
+- 新增执行剖析功能，当数据源版本为 OceanBase 4.2.4 或更高时，支持查询执行过程中或已完成的 SQL 执行详情
```
[the remark this answers is not in the document]
Comment (LuckyPickleZZ) [reply CHANGELOG-zh-CN.md:19]: impressive 👏👏

File: CHANGELOG-zh-CN.md [hunk 9]
```diff
@@ -1,5 +1,72 @@
...
+SQL 开发
+
+- SQL 执行时支持查看执行进度，包括：总执行条数、当前执行条数及当前正在执行 SQL 的 trace id；支持实时查看已完成的执行结果
+- 支持以图形格式查看 OceanBase 的逻辑 SQL 执行计划
+
+### 易用性改进
+
+- 数据清理任务支持编辑任务配置
+- 数据源模块，批量导入支持 mysql、oracle 和 doris 数据源
+
```
Comment (Jane201510) [inline CHANGELOG-zh-CN.md:33]: OB 版本适配：全功能适配 OB 4.2.4、OB 4.3.1、OB 4.3.2。

**Review by Jane201510 — APPROVED** [review]

**Review by yizhouxw — APPROVED** [review]

File: CHANGELOG-zh-CN.md [hunk 10]
```diff
@@ -1,5 +1,72 @@
...
+### 功能变化
+
+变更风险管控
+
+- 新增表级别权限管控，允许项目成员对不同表拥有不同操作权限，包括对表的查询、变更和导出操作，进一步加强管控协同能力
+
+会话管理
+
+- 关闭会话/查询支持更多场景
```
[the remark this answers is not in the document]
Comment (LuckyPickleZZ) [reply CHANGELOG-zh-CN.md]: updated

File: CHANGELOG-zh-CN.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (LuckyPickleZZ) [reply CHANGELOG-zh-CN.md]: done~

File: CHANGELOG-zh-CN.md [hunk 11]
```diff
@@ -1,5 +1,72 @@
...
+
+### 易用性改进
+
+- 数据清理任务支持编辑任务配置
+- 数据源模块，批量导入支持 mysql、oracle 和 doris 数据源
+
+### 缺陷修复
+
+数据生命周期管理
+- 如果 syncTableStructure 配置关闭，则不比较表结构 [#3014](https://github.com/oceanbase/odc/pull/3014)
```
[the remark this answers is not in the document]
Comment (LuckyPickleZZ) [reply CHANGELOG-zh-CN.md]: done

File: CHANGELOG-zh-CN.md [hunk 12]
```diff
@@ -1,5 +1,72 @@
...
+
+变更风险管控
+- 自动授权规则对 LoginSuccess 事件不生效 [#3003](https://github.com/oceanbase/odc/pull/3003)
+
+导入导出
+- 桌面版模式下，如果清理了元数据库但未清理工作空间，则再次发起导入任务时可能会存在脏数据文件 [#3006](https://github.com/oceanbase/odc/pull/3006)
```
[the remark this answers is not in the document]
Comment (LuckyPickleZZ) [reply CHANGELOG-zh-CN.md]: done

File: CHANGELOG-zh-CN.md (hunk 5 above)
Comment (LuckyPickleZZ) [reply CHANGELOG-zh-CN.md]: done

File: CHANGELOG-zh-CN.md (hunk 6 above)
Comment (LuckyPickleZZ) [reply CHANGELOG-zh-CN.md]: updated

File: CHANGELOG-zh-CN.md (hunk 7 above)
Comment (LuckyPickleZZ) [reply CHANGELOG-zh-CN.md]: done

File: CHANGELOG-zh-CN.md (hunk 9 above)
Comment (LuckyPickleZZ) [reply CHANGELOG-zh-CN.md:42]: thanks, it's added~

**Pull request merged**